spec/integration/cli/server_spec.rb

```diff
@@ -364,4 +364,17 @@ def listing
       end
     end
   end
+
+  context "without email" do
+    it "returns page" do
+      with_project do
+        replace "config/environment.rb", "delivery :test", ""
```

Comment:
@davydovanton I'm afraid this isn't enough. If you look at the original [bug report](https://github.com/hanami/hanami/issues/740#issue-211972961), there was more things removed. Specifically, all the `mailer` blocks were stripped from the original configuration. We should reproduce that scenario.

There are other examples here that show how to rewrite an entire file. That could be a solution.